app/src/main/java/org/thoughtcrime/securesms/ui/Util.kt

```diff
@@ -18,6 +19,14 @@ fun Context.createThemedComposeView(content: @Composable () -> Unit): ComposeVie
     setThemedContent(content)
 }
 
+fun Context.getSubbedString(stringId: Int, vararg substitutionPairs: Pair<String, String>): CharSequence {
+    val phrase = Phrase.from(this, stringId)
+    for ((key, value) in substitutionPairs) {
+        phrase.put(key, value)
+    }
+    return phrase.format()
```

Comment:
Maybe we should return `format().toString()` to avoid repeating it everywhere else?
Some cases DO need the charSequence though.
So maybe you could rename this function as `Context.getString` to keep it simple
And add another one called `Context.getCharSequence` that stops at `format` without the `toString`

Reply:
`Context.toString` shadows a system call with formatting arguments, so I have it as `Context.getSubbedString` atm.

Also, I've left it return a CharSequence because as you say, sometimes that's exactly what we need, and I then I `toString()` the result. We can't have two `getSubbedString` methods, one returning CharSequence and one returning String because there are implicit casts which make them non-uniquely-identifiable.

I'll proceed as mentioned, but if you feel strongly we can discuss further :)